ggshield/verticals/sca/output/json_handler.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Dict, cast
+
+from ggshield.verticals.sca.collection.collection import (
+    SCAScanAllVulnerabilityCollection,
+    SCAScanDiffVulnerabilityCollection,
+)
+from ggshield.verticals.sca.output.handler import SCAOutputHandler
+from ggshield.verticals.sca.output.schemas import (
+    JSONSCAScanAllOutputSchema,
+    JSONSCAScanDiffOutputSchema,
+)
+
+
+class SCAJsonOutputHandler(SCAOutputHandler):
+    def _process_scan_all_impl(self, scan: SCAScanAllVulnerabilityCollection) -> str:
+        # We precise is not None as __bool__ have been overriden for pygitguardian
+        # Base class
+        scan_dict = self.create_scan_all_dict(scan)
+        schema = JSONSCAScanAllOutputSchema()
+        serialized_result = schema.load(scan_dict)
+        text = schema.dumps(serialized_result)
```

Comment:
is there a reason why we can't directly do `text = JSONSCAScanAllOutputSchema().dumps(scan_dict)` ?

Reply:
It throws an error because a field is a datetime, and while dumping it tries to use a datetime function on a str.